README.md

```diff
@@ -46,6 +46,7 @@ So we need to install these packages first:
     # Ubuntu 10.4+
     sudo apt-get install protobuf-c-compiler
     sudo apt-get install libprotobuf-c0-dev
```

Comment:
Yes, you're right here. After updating dependencies, the combination was `libprotobuf-c-dev` + `libprotoc-dev`.